packages/yumemi_lints/lib/dart/2.18/all.yaml

```diff
@@ -88,7 +88,7 @@ linter:
     - noop_primitive_operations # categories: style
     - null_check_on_nullable_type_parameter # categories: style
     - null_closures # categories: error-prone
-    - omit_local_variable_types # incompatibles: always_specify_types # categories: style
+    - omit_local_variable_types # incompatibles: always_specify_types,specify_nonobvious_local_variable_types # categories: style
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey) 
`specify_nonobvious_local_variable_types` is a rule that will be added in Dart 3.6.0.
https://github.com/dart-lang/sdk/commit/5b7c600d8b66a5a3236607d3f4569f150053c7a5